components/authz-service/storage/postgres/migration/sql/52_add_db_id_members.up.sql

```diff
@@ -0,0 +1,156 @@
+BEGIN;
+ALTER TABLE iam_policy_members
+    DROP CONSTRAINT iam_policy_members_member_id_fkey;
+ALTER TABLE iam_members
+    DROP CONSTRAINT iam_members_pkey;
+ALTER TABLE iam_members
+    ADD COLUMN db_id SERIAL PRIMARY KEY UNIQUE;
+ALTER TABLE iam_policy_members RENAME COLUMN member_id TO member_temp_id;
+ALTER TABLE iam_policy_members
+    ADD COLUMN member_id INTEGER REFERENCES iam_members (db_id) ON DELETE CASCADE;
+UPDATE
+    iam_policy_members ipm
+SET
+    member_id = (
+        SELECT
+            db_id
+        FROM
+            iam_members
+        WHERE
+            id = ipm.member_temp_id);
+ALTER TABLE iam_policy_members
+    DROP COLUMN member_temp_id;
+ALTER TABLE iam_members
+    ADD UNIQUE (id, name);
+ALTER TABLE iam_policy_members
+    ADD UNIQUE (member_id, policy_id);
+-- add helper for member db_id lookup
+CREATE FUNCTION member_db_id (_id UUID)
+    RETURNS INTEGER
+    AS $$
+    SELECT
+        db_id
+    FROM
+        iam_members
+    WHERE
+        id = _id;
+$$
+LANGUAGE SQL;
+CREATE OR REPLACE FUNCTION projects_match (_projects TEXT[], _projects_filter TEXT[])
```

Comment:
❓ why did this function have to be changed? I don't see any IDs in it... 🤔

Reply:
🤔